R/NielsonKuparuk.R

```diff
@@ -0,0 +1,278 @@
+               attributeList = attributeList1)
+
+dt_KupTR <- new('dataTable',
+                 entityName = "KupTR.csv",
+                 entityDescription = "Toolik River at mouth of Kuparuk River" ,
+                 physical = physicalKup2TQ,
+                 attributeList = attributeList1)
+dt_KupBWH <- new('dataTable',
+                entityName = "KupBWH.csv",
+                entityDescription = "Kuparuk River below White Hills creek confluence" ,
+                physical = physicalKup3BQ,
+                attributeList = attributeList1)
+dt_KupBOK <- new('dataTable',
+                 entityName = "KupBOK.csv",
+                 entityDescription = "Kuparuk River below Old Kuparuk" ,
+                 physical = physicalKup3TQ,
+                 attributeList = attributeList1)
+dt_KupBA <- new('dataTable',
+                 entityName = "KupBA.csv",
+                 entityDescription = "Kuparuk River below aufeis" ,
+                 physical = physicalKup4Q,
+                 attributeList = attributeList1)
+
+dt_KupAA <- new('dataTable',
+                entityName = "KupAA.csv",
+                entityDescription = "Kuparuk River above aufeis" ,
+                physical = physicalKup5Q,
+                attributeList = attributeList1)
+dt_KupBIC <- new('dataTable',
+                entityName = "KupBIC.csv",
+                entityDescription = "Kuparuk River below Imnavait Creek confluence" ,
+                physical = physicalKup8Q,
+                attributeList = attributeList1)
+
+dt_KupRoadQ <- new('dataTable',
+                 entityName = "KupRoadQ.csv",
+                 entityDescription = "Kuparuk River above Dalton Highway" ,
+                 physical = physicalKup9Q,
+                 attributeList = attributeList1)
+
+eml@dataset@dataTable <- c(dt_IC_Q, dt_IC_Weir, dt_KupBWH, dt_KupBOK, dt_KupAA, dt_KupBA, dt_KupATR, dt_KupTR, dt_KupBIC, dt_KupRoadQ)
+
+#Edit names
+eml@dataset@dataTable@.Data[[10]]@entityName
+eml@dataset@dataTable@.Data[[10]]@physical@.Data[[1]]@objectName <- "KupRoadQ.csv"
+
+#Validate and write eml
+eml_validate(eml)
+write_eml(eml, "/home/mnguyen/Neilson/Neilson_Metadata.xml")
+
+#Update package
+dataPid <- c(data_pidIC_WeirQ, data_pidICKUP_Q , data_pidKup2AQ , data_pidKup2TQ , data_pidKup3BQ, data_pidKup3TQ , data_pidKup4Q , data_pidKup5Q , data_pidKup8Q , data_pidKup9Q) 
+
+ids <- get_package(mn, "urn:uuid:6674785c-f684-4cfc-922e-35630d18ed1a")
+publish_update(mn,
+               metadata_pid = ids$metadata,
+               resource_map_pid = ids$resource_map,
+               metadata_path = "/home/mnguyen/Neilson/Neilson_Metadata.xml",
+               data_pids = ids$data,
+               check_first = TRUE,
+               use_doi = TRUE,
+               public = TRUE)
+
+#Edit system metadata
+system <- getSystemMetadata(mn, "doi:10.18739/A2K87V")
+system@fileName <- "KupRoadQ.csv"
+updateSystemMetadata(mn, "urn:uuid:955e80f1-f1d9-4bee-8cf9-2393bcb9ece6", system)
```

Comment:
in case you don't know about `set_file_name`, you can use it to do all this in one step